DevOps/format_all_files.ps1

```diff
@@ -0,0 +1,43 @@
+# Function to traverse directories with exclusions
```

Comment:
Let's merge it, but it is only for Windows. I'm working on Mac, so I cannot use it. I think it would be better to stick to some cross-platform solution. We can use shell scripts with powershell wrappers or use Python